examples/reference/panes/HoloViews.ipynb

```diff
@@ -119,18 +207,205 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "widgets = hv_panel[1]\n",
+    "plot = df.hvplot.scatter(x=\"group\", y=\"value\")\n",
+    "pn.pane.HoloViews(plot, backend='matplotlib', sizing_mode=\"stretch_width\", height=300)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Please note that in a *server context* you might have to set the matplotlib backend like below depending on your setup.\n",
+    "\n",
+    "```python\n",
+    "import matplotlib\n",
+    "matplotlib.use('agg')\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can make your matplotlib plot stretch responsively by ???? **DON'T KNOW HOW TO. PLEASE HELP**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plotly\n",
+    "\n",
+    "To use the 'plotly' plotting backend you will need to run `hv.extension(\"plotly\")` to configure the 'plotly' backend.\n",
+    "\n",
+    "If you are using `hvPlot` you can use `hvplot.extension(\"plotly\")` instead."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hvplot.extension(\"plotly\")\n",
+    "\n",
+    "plot = df.hvplot.scatter(x=\"group\", y=\"value\", height=300, responsive=True)\n",
+    "pn.pane.HoloViews(plot, backend='plotly', height=300)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lets change the default backend back to 'bokeh'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hvplot.extension(\"bokeh\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Dynamic\n",
+    "\n",
+    "You can change the plotting backend dynamically via a widget too."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot = df.hvplot.scatter(x=\"group\", y=\"value\", height=300, responsive=True, title=\"Try changing the backend\")\n",
```

Comment:
This example is the demonstrating the core of HoloViews to me. That I can easily output to bokeh, matplotlib and plotly.

Unfortunately it does not really work.

- Only the bokeh plot has the `title`.
- The matplotlib plot does not stretch_width like the other plots
- The plotly plot does not stretch_width. And its height is higher than the 300px specified both with hvPlot and HoloViews.


https://github.com/holoviz/panel/assets/42288570/ec3bd862-e374-476d-ae2c-ad403cc26653

Reply:
Indeed, this would be very nice. Thanks for opening this issue: https://github.com/holoviz/hvplot/issues/1236 where we will track this going forward.